reconcile/aws_ecr_image_pull_secrets.py

```diff
@@ -0,0 +1,52 @@
+import base64
+import json
+import logging
+
+import reconcile.queries as queries
+import utils.vault_client as vault_client
+
+from utils.aws_api import AWSApi
+
+QONTRACT_INTEGRATION = 'aws-ecr-image-pull-secrets'
+
+
+def enc_dec(data):
```

Comment:
can you explain what this is for?

Reply:
this is to handle TypeErrors from base64 encoding and decoding of strings (and thank you to python3).

Reply:
got it, base64encode expects a bytes-like object...